okhttp/src/main/kotlin/build/buf/connect/okhttp/OkHttpStream.kt

```
@@ -193,7 +193,7 @@ internal class PipeDuplexRequestBody(
                 bufferedSink.writeAll(buffer)
                 bufferedSink.flush()
             }
-        } catch (e: Throwable) {
+        } finally {
```

Don't swallow these errors anymore and let the caller handle errors